.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: weekly
+      day: monday
```

Comment:
I don't presently care what the schedule is, so long as it is relatively frequent.  This is a suggestion, as I've found updates more than weekly to be busy in a large project.  This repo may be small enough that it makes sense to do updates daily or continuously (e.g. as soon as an upstream issues them).